src/Game.java

```diff
@@ -0,0 +1,46 @@
+import java.util.LinkedList;
+
+public class Game {
+    Deck deck = new Deck();
+    String player1Name;
+    String player2Name;
+    String player3Name;
+    String player4Name;
+    LinkedList<Player> players = new LinkedList<>();
+    int startingHandSize;
+    int currentHandSize;
+
+
+    public void addPlayers() {
+        players.add(new Player(player1Name, 'N'));
+        players.add(new Player(player2Name, 'E'));
+        players.add(new Player(player3Name, 'S'));
+        players.add(new Player(player4Name, 'W'));
+    }
+
+    public void dealCards(Deck deck, int currentHandSize) {
+        for (Player player : players) {
+            for (int j = 0; j < currentHandSize; j ++) {
+                player.hand.add(deck.cards.poll());
```

Comment:
I would actually prefer if you wrote a function in `Player` called `addCardToHand`. That way we keep the implementation of the hand separate from the way other classes interface with it